utils/create-html-certificate.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const fs = require('fs');
+const path = require('path');
+const { generateHtml } = require('@s1seven/schema-tools-generate-html');
+
+(async function () {
+  // incase you run directly with node from utils, uncomment the following line:
+  // process.chdir('../');
+
+  const argv = yargs(hideBin(process.argv))
+    .usage('Usage: $0 -s [schemaPath] -o [outputPath] -t [translationsPath] -T [templatePath]')
+    .options({
+      certificatePath: {
+        description: 'Path to the schema',
+        demandOption: true,
+        type: 'string',
+        coerce: (certificatePath) => {
+          if (!fs.existsSync(path.resolve(certificatePath))) {
+            throw new Error('Certificate file path does not exist.');
```

Comment:
I don't think we need this file anymore right ?

Reply:
This is one of the tools we made yesterday using yargs, are you sure we don't need it?

Reply:
My mistake, the culprit seems to be [packages/generate-coa-pdf-template/utils/create-schema-interfaces-interactive.js](https://github.com/s1seven/schema-tools/pull/116/files#diff-773746113e99e4b7615605e4bbb8b2636fa9d8ae297543cd93da9c26eef2cdf9)

Reply:
No problem, that's done 👍🏼